frontend/components/AddSite/TemplateSiteList/templateSite.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 
+import { getSafeRepoName } from '../../../util';
+
 const propTypes = {
   name: React.PropTypes.string.isRequired,
-  owner: React.PropTypes.string.isRequired,
```

Comment:
`owner` appeared unused, but I did see reference to using `props.defaultOwner`, so I renamed `owner` to `defaultOwner`